examples/convecting_plankton.jl

```diff
@@ -1,88 +1,148 @@
+# which sunlight attenuates away from the surface, and ``m`` is the mortality rate
+# of phytoplankton due to viruses and grazing by zooplankton. We use Oceananigans'
+#  `Forcing` abstraction to implement the phytoplankton dynamics described by the right
+# side of the phytoplankton equation above.
+#
+# This example demonstrates
+#
+#   * How to use a user-defined forcing function to
+#     simulate the dynamics of phytoplankton growth in sunlight
+#     and grazing by zooplankton.
+#   * How to set time-dependent boundary conditions.
 #   * How to use the `TimeStepWizard` to adapt the simulation time-step.
 #   * How to use `AveragedField` to diagnose spatial averages of model fields.
 #
 # ## The grid
 #
-# We use a two-dimensional grid with 128² points and 2 m grid spacing:
+# We use a two-dimensional grid with 64² points and 1 m grid spacing:
 
 using Oceananigans
 
-grid = RegularCartesianGrid(size=(128, 1, 128), extent=(64, 1, 64))
+grid = RegularCartesianGrid(size=(64, 1, 64), extent=(64, 1, 64))
 
 # ## Boundary conditions
 #
-# We impose buoyancy loss at the surface with the buoyancy flux
+# We impose a surface buoyancy flux that's initially constant and then decays to zero,
 
-Qb = 1e-8 # m³ s⁻²
-nothing # hide
+using Oceananigans.Utils
 
-# Note that a _positive_ flux at the _top_ boundary means that buoyancy is
-# carried _upwards_, out of the fluid. This reduces the fluid's buoyancy 
-# near the surface, causing convection.
-#
-# The initial condition consists of the constant buoyancy gradient
+buoyancy_flux(x, y, t, p) = p.initial_buoyancy_flux * exp(-t^4 / (24 * p.shut_off_time^4))
 
-N² = 1e-6 # s⁻²
-nothing # hide
+buoyancy_flux_parameters = (initial_buoyancy_flux = 1e-8, # m² s⁻³
+                                    shut_off_time = 2hours)
 
-# which we also impose as a boundary condition at the bottom.
+buoyancy_flux_bc = BoundaryCondition(Flux, buoyancy_flux, parameters = buoyancy_flux_parameters)
 
-buoyancy_bcs = TracerBoundaryConditions(grid, top = BoundaryCondition(Flux, Qb),
-                                              bottom = BoundaryCondition(Gradient, N²))
+# The fourth power in the argument of `exp` above helps keep the buoyancy flux relatively
+# constant during the first phase of the simulation. We produce a plot of this time-dependent
+# buoyancy flux for the visually-oriented,
 
-# ## Forcing: the growing and grazing of phytoplankton
+using Plots, Measures
+
+time = range(0, 12hours, length=100)
+
+flux_plot = plot(time ./ hour, [buoyancy_flux(0, 0, t, buoyancy_flux_parameters) for t in time],
+                 linewidth = 2, xlabel = "Time (hours)", ylabel = "Surface buoyancy flux (m² s⁻³)",
+                 size = (800, 300), margin = 5mm, label = nothing)
+
+# The buoyancy flux effectively shuts off after approximately 6 hours of simulation time.
 #
-# We add a forcing term to the plankton equation that crudely models:
+# !!! info "The flux convention in Oceananigans.jl"
```

Comment:
Admonition? Something's missing.

Reply:
It looks fine to me:

![image](https://user-images.githubusercontent.com/15271942/98441637-9a0f2380-20cd-11eb-9246-3a0eff517087.png)

What's wrong with it?

Reply:
Nothing. I think I just mixed  up the PR. The problem was in the Langmuir one! :)